yarn.lock

```diff
@@ -13153,7 +13143,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"chownr@npm:^1.1.1":
+"chownr@npm:^1.1.1, chownr@npm:^1.1.4":
```

Comment:
This is an example of a "deduped" dependency in `yarn.lock` -- anything requesting v 1.1.1 or 1.1.4 will both resolve to 1.1.4, thus avoiding having extra copies of `chown` on the file system